examples/parquet_analytics_demo.ipynb

```diff
@@ -0,0 +1,476 @@
+    "storage_download(container, \"organization.parquet\", phdi_organizations)\n",
+    "storage_download(container, \"composition.parquet\", phdi_compositions)\n",
+    "storage_download(container, \"patient.parquet\", phdi_patients)\n",
+    "\n",
+    "# Notice that we can read-in only a subset of columns if we so choose\n",
+    "# i.e. It's not necessary to fully read the file to memory\n",
+    "conditions = read_parquet(phdi_conditions, col_select = c(\"patient_id\", \"onset_date\"))\n",
+    "organizations = read_parquet(phdi_organizations)\n",
+    "compositions = read_parquet(phdi_compositions)\n",
+    "patients = read_parquet(phdi_patients)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a83320d1",
+   "metadata": {},
+   "source": [
+    "We've now created local copies of all data that maps into the ECR schema--we have resources for Patients, Conditions, Organizations, and Compositions (between patients and organizations). Importantly, we've stored all of this data into `tempfiles` which will persist _only_ as long as our current R session. That way, when we're finished with analysis, the files will be cleared, leaving no chance for PII to linger. We'll use these file for the remainder of our analysis demo, noting that pulling files down this way allows analysts to experiment freely on data without any risk of corrupting the source of truth.\n",
+    "\n",
+    "Before getting into some demo analytics problems, we need to do a bit of data modification and format. If we try to visualize the `patient` data frame, we'll get an error describing that the expected length of a display string should be one value, but a different value was encountered by the print formatting function (the exact values themselves aren't important, it's the discrepancy in the numbers that breaks the display):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1d4e9a0c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# This call WILL result in an error\n",
+    "head(patients)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0749f6fb",
+   "metadata": {},
+   "source": [
+    "This is because the `phone_number` column is nost only a nested list, but because the information stored within that column is of variable length (because some patients have phone numbers on file while others do not). Let's format this column to line up with what we'd like our data frame to be: a singular, flattened entity on which we can do joins and perform analysis."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e56dbcf0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Flatten the list column to a series of regular columns\n",
+    "patients <- patients %>% unnest(phone_number)\n",
+    "patients <- subset(patients, select = -c(system, use))\n",
+    "patients <- rename(patients, phone_number = value)\n",
```

Comment:
nit: This could be more "r-thonic" (that's not a thing like pythonic is, but whatever) if you changed it to:

```r
patients <- patients %>%
    unnest(phone_number) %>%
    subset(select=-c(system, use)) %>%
    rename(phone_number=value)
```

Users who are familiar with the tidyverse are likely more accustomed to seeing a sequence of data wrangling methods chained together with the pipe like this. There's nothing wrong with how you've done it, but it could increase readability for the users. Do with this information what you will.

Reply:
Oh yup you're right, I did that elsewhere so I don't know why I didn't do that here